modules/ROOT/pages/index.adoc

```diff
@@ -2,29 +2,111 @@
+
+For example, you can create a monitor that runs a test twice daily to check whether an API that handles requests to a database is returning the correct data within the expected response time. The test can use the HTTP GET method to send a request to the endpoint at which the API is awaiting such requests. You can set two schedules: one to run the test at 9 in the morning, and another to run the test at 3 in the afternoon. You can also set the monitor to notify you via an email message, a Slack post, or another method through one of the supported third-party notification tools, when a test execution fails. You also get a summary of the test results whether or not a test execution fails, and can look up the results of past executions of the test.
+
+Scenarios that are more complicated are also possible. A test can run against multiple endpoints, making requests with any HTTP methods.
+
+=== Fundamentals of Tests
+
+Each monitor includes one test. A test consists of:
+
+* One or more combinations of HTTP methods and endpoints
++
+For example, in a single test, you might send a request to the endpoint `\https://deckofcardsapi.com/api/deck/new/shuffle/?deck_count=1` by using the HTTP method GET and a request to the endpoint `\https://deckofcardsapi.com/api/deck/<deck_ID>/shuffle/` also by using the GET method.
++
+In a single test, you can send requests to the endpoints in public APIs, private APIs, or a mix of both.
++
+You can create dependencies between tests of endpoints. For example, if in your application network a request to endpoint B must be preceded by a successful request to endpoint A, you can write a test that sends a request to endpoint B only if the request to endpoint A is successful.
+
+* One or more assertions
++
+By default, the assertion for a test is "Status code must equal 200". However, you can change that assertion and add more. For example, you might add the assertion "Response header Cache-Control must equal `max-age=604800`".
+
+* Optional HTTP request headers
++
+By default, no headers are included in requests sent to endpoints. However, you can include any of the headers that AFM supports.
+
+Tests are executed from either public locations or private locations.
+
+* A _public location_ is a region (which you can think of as a resource pool) that is shared with other MuleSoft customers.
++
+Public locations can run only tests of public APIs.
+
+* A _private location_ is a worker that runs in a CloudHub environment that is associated with an instance of Anypoint Virtual Private Cloud.
++
+Private locations can run tests of public APIs, private APIs, or mixes of both.
+
+=== Fundamentals of Schedules
+
+A schedule initiates a worker, at intervals that you specify, to run the test for a monitor from a particular location, whether the location is public or private. Each schedule is associated with one test.
```

Comment:
The schedule initiates a worker or the test run? Clarify by using simpler sentence syntax.